### ISSUE TITLE
Doesn't handle bends in vertical axis properly.

### ISSUE THREAD

**Issue by ZackMercury** [nickjanssen/PatrolJS#5]
![image](https://user-images.githubusercontent.com/10474828/27673922-0ec0f4cc-5cac-11e7-8e0d-06ab89dd8f0c.png)
If you put begin node straight against the stairs, and the end node on the platform, it does this (screenshot). It should be putting dots down the stairs and up the stairs.

**Comment by sandip-narwade** [comment]
It showing only path, but sphere will come on stairs

**Comment by ZackMercury** [comment]
> It showing only path, but sphere will come on stairs

No, it goes by the path. Anyway, even if it would do that, the expected output of the algorithm is to have correct **path** between 2 points.

**Comment by sandipJava** [comment]
@ZackMercury  I need a help in Obj and patrol.JS

**Comment by nickjanssen** [comment]
Check out [three-pathfinding](https://github.com/donmccurdy/three-pathfinding), maybe it'll solve your issue.

**Comment by sandipJava** [comment]
@nickjanssen  @ZackMercury , how to add nodes in straight path.

Hello Sir, This question isn’t related to threeJs, thats why I am sending personal msg, bCoz you worked on patrol.js. Can you tell me how to increase node in patrol.js. suppose if turning is path it is taking more nodes(green box) but if path is straight its taking only one node, How to increase nodes in straight path. See Image. [Image](upload://2H6VlIB8au4ljo5NBhqO3CEE8HQ.png)
